pkg/util/fs_util_test.go

```diff
@@ -58,7 +58,6 @@ func Test_DetectFilesystemWhitelist(t *testing.T) {
 		{"/dev", false},
 		{"/dev/pts", false},
 		{"/sys", false},
-		{"/var/run", false},
```

Comment:
This test is little confusing. Seems like we are testing 
- whitelist before `AddToWhitelist` 
- whitelist after `AddToWhitelist`

Can we break this into 2 tests or only one after `AddToWhitelist` should suffice.

Reply:
Ideally, it would be great if we convert this to Test Table driven with test cases
- no addiional whitelists
- 1 additional whitelist
- 1 or more additional whitelist

Reply:
done :+1: